data_structures/binary_tree/binary_tree_traversals.py

```diff
@@ -1,24 +1,16 @@
 # https://en.wikipedia.org/wiki/Tree_traversal
+from dataclasses import dataclass
 
 
-class Node:
-    """
-    A Node has data variable and pointers to its left and right nodes.
-    """
-
-    def __init__(self, data):
-        self.left = None
-        self.right = None
-        self.data = data
+@dataclass
+class Node(object):

```

Comment:
```suggestion
class Node:
```
By default, we are subclassing `object` so there's no point in including it.